Update kiva

aaron-boranian opened the pull request:
Update to CSE.exe with recent Kiva updates.
Update to CSE.exe v0.923.1 (3-7-25).

Make soil properties used by Kiva consistent with [EnergyPlus test suite](https://github.com/bigladder/ep-resnet-ref).

Fix reporting of basement infiltration load in Detailed exports file and total building heating load.

aaron-boranian committed:
Update CSE.exe to v0.905.0.

aaron-boranian committed:
Update to CSE.exe with recent Kiva updates.

- CSE.exe can be found here: https://github.com/cse-sim/cse/actions/runs/2110440035
- Update seasonal heating probes reported in "Seasonal Temperatures.csv".

aaron-boranian committed:
Make soil properties used by Kiva consistent with EnergyPlus test suite.

- Soil conductivity = 1.0 Btu/hr-ft-F (1.73 W/m-K)
- Soil specific heat = 0.1 Btu/lb-F (419 J/kg-K)
- Soil density = 115 lb/ft3 (1842.12 kg/m3)
- These are all default values for each property in both CSE and EnergyPlus.

aaron-boranian committed:
Fix reporting of basement infiltration load in Detailed exports file and total building heating load.

- Was previously reporting main zone infiltration load instead of basement.
- Was previously reporting only main zone heating load, which explains low results for Section 7 cases with basement (L322, L324).

nealkruis commented on base.pxt:
```diff
@@ -940,11 +944,16 @@ EXPORT
 // Heating seasonal data
-  EXPORTCOL colHead="Rsysres Heating [Btu]" colVal=@rsysres["Main Zone HVAC System"].S.qh
-  EXPORTCOL colHead="Rsysres Heating Hours On [hr]" colVal=select(@rsysres["Main Zone HVAC System"].S.qh > 0, @rsysres["Main Zone HVAC System"].S.hrsOn ,default 0)
+//  EXPORTCOL colHead="Rsysres Heating [Btu]" colVal=@rsysres["Main Zone HVAC System"].S.qh
```
Is there a reason to comment this out instead of removing it entirely?

aaron-boranian replied:
In case heating seasonal energy is important for QAQC steps in the future. I could create a new header parameter to toggle this on/off instead.

nealkruis commented on base.pxt:
```diff
@@ -940,11 +944,16 @@ EXPORT
   <% if foundation_type == "BASEMENT" %>
   EXPORTCOL colHead="Basement Temp [F]" colVal=@znres["Basement Zone"].S.tAir
-  EXPORTCOL colHead="Basement Heating Seasonal Temp [F]" colVal=select(@rsysres["Main Zone HVAC System"].S.qh > 0, @znres["Basement Zone"].S.tAir * @rsysres["Main Zone HVAC System"].S.hrsOn, default 0)
+//  EXPORTCOL colHead="Basement Heating Seasonal Temp [F]" colVal=select(@rsysres["Main Zone HVAC System"].S.qh > 0, @znres["Basement Zone"].S.tAir * @rsysres["Main Zone HVAC System"].S.hrsOn, default 0)
```
Is there a reason to comment this out instead of removing it entirely?

aaron-boranian replied:
In case basement heating seasonal temperature is important for QAQC steps in the future. I could create a new header parameter to toggle this on/off instead.

aaron-boranian committed:
Add header parameter to generate seasonal temperature outputs.

- These outputs are used for seasonal DSE calculations related to ASHRAE Standard 152
- Replace outputs using S.qh (incorrect probe, used with @ahRes) with S.qhNet (correct probe, net total of all other heating probes)
- Adjust indenting of last if/end block.

aaron-boranian committed:
Update CSE to v0.908.

- This is the latest public release that includes Kiva fixes used in temporary Github action release in [commit 10538b2](https://github.com/bigladder/cse-resnet-ref/pull/1/commits/10538b221e047bec30fbc9ef91c1bbaf1d78ddd4)

nathancoliver committed:
Update CSE to v0.923.1.
